test/e2e/testnet/defaults.go

```diff
@@ -10,7 +10,7 @@ var DefaultResources = Resources{
 }
 
 const (
-	TxsimVersion = "v3.3.1"
+	TxsimVersion = "b5a8e7c" // TODO: use a proper version, this is one built on the sdk 50 fork
```

Comment:
Another TODO that needs to be tracked in a follow up issue

Reply:
cc. @chatton

Reply:
this should fall under the list I'll created as follow ups for #82 this should be a tagged celestiaorg version.